src/templates/view/TemplateShortcutListener.ts

```diff
@@ -31,7 +29,8 @@ class TemplateShortcutListener {
 	}
 
 	// add this event listener to handle quick selection of templates inside the editor
-	handleKeyDown(event: Event) {
+	handleKeyDown(event: KeyboardEvent) {
+		console.log("handling keydown!", event)
```

Comment:
Do we want this log? Genuinely don't know.

Reply:
oh no, I forgot about that one.